Cabal/Distribution/Simple/Program/Strip.hs

```diff
@@ -53,6 +56,17 @@ stripLib verbosity (Platform _arch os) conf path = do
                -- libraries with lots identically named modules. See
                -- #1784.
                return()
+    Linux | arch == I386 ->
+      -- Versions of 'strip' on 32-bit Linux older than 2.18 are
+      -- broken. See #2339.
+      let (Just okVersion) = simpleParse ">= 2.18"
```

Comment:
D'oh, of course. I thought cabal used a special opaque `Version`. Patch incoming